packages/assets-controllers/src/NftController.test.ts

```diff
@@ -3927,5 +3927,105 @@ describe('NftController', () => {
 
       expect(spy).toHaveBeenCalledTimes(1);
     });
+
+    it('should call getNftInformation only one time per interval', async () => {
+      const { nftController, triggerPreferencesStateChange } =
+        setupController();
+      const { selectedAddress } = nftController.config;
+      const spy = jest.spyOn(nftController, 'updateNft');
+      const testNetworkClientId = 'sepolia';
+      await nftController.addNft('0xtest', '3', {
+        nftMetadata: { name: '', description: '', image: '', standard: '' },
+        networkClientId: testNetworkClientId,
+      });
+
+      sinon
+        .stub(nftController, 'getNftInformation' as keyof typeof nftController)
```

Comment:
We are no longer stubbing private functions. You can refer to PR [#4310](https://github.com/MetaMask/core/pull/4310) for details on how this has been achieved and to see the new approach.

Reply:
Right! Thank you, i fixed that 🙏